qdrant_client/qdrant_client.py

```diff
@@ -53,57 +55,77 @@ def __init__(self,
                  port=6333,
                  grpc_port=6334,
                  prefer_grpc=False,
-                 https=False,
+                 https=None,
+                 api_key=None,
                  **kwargs):
         self._prefer_grpc = prefer_grpc
         self._grpc_port = grpc_port
         self._host = host
         self._port = port
+
+        self._https = https
+        self._api_key = api_key
+
+        http2 = kwargs.pop("http2", False)
+        grpc_headers = {}
+        rest_headers = kwargs.pop("headers", {})
+        if api_key is not None:
+            if self._https is False:
+                warnings.warn("Api key is used with unsecure connection.")
```

Comment:
Did you check if the load balancer accept HTTP traffic with an API key?

Reply:
our cloud is https only